Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -1016,13 +1017,220 @@ def set_user_command(args):
+            try:
+                cmd_url = "/ipDestinationGroups"
+                if exclude_type != '':
+                    cmd_url += f"?excludeType={exclude_type}"
+                responses = http_request('GET', cmd_url).json()
+            except json.decoder.JSONDecodeError as exp:
+                return_error(f'Failed to execute zscaler-list-ip-destination-group command. Error: {str(exp)}')
+            contents = list()
+            for response in responses:
+                content = {
+                    'ID': int(response.get('id', '')),
+                    'Name': response.get('name', ''),
+                    'Type': response.get('type', ''),
+                    'Description': response.get('description', ''),
+                    'Addresses': response.get('addresses', []),
+                    'IpCategories': response.get('ipCategories', []),
+                    'Countries': response.get('countries', [])
+                }
+                contents.append(content)
+            entry = {
+                'Type': entryTypes['note'],
+                'Contents': contents,
+                'ContentsFormat': formats['json'],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable': tableToMarkdown(("IPv4 Destination groups"
+                                                  + f" ({len(contents)})"),
+                                                 contents),
+                'EntryContext': {'Zscaler.IPDestinationGroup': contents},
+            }
+    else:
+        try:
+            cmd_url = f"/ipDestinationGroups/{ip_group_id}"
+            response = http_request('GET', cmd_url).json()
+        except json.decoder.JSONDecodeError as exp:
+            return_error(f'Failed to execute zscaler-list-ip-destination-group command. Error: {str(exp)}')
+        content = {
+            'ID': int(response.get('id', '')),
+            'Name': response.get('name', ''),
+            'Type': response.get('type', ''),
+            'Description': response.get('description', ''),
+            'Addresses': response.get('addresses', []),
+            'IpCategories': response.get('ipCategories', []),
+            'Countries': response.get('countries', [])
+        }
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': content,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown("IP Destination group", content),
+            'EntryContext': {'Zscaler.IPDestinationGroup': content},
+        }
+    return entry
+
+
+def edit_ip_destination_group(args: dict):
+    payload = dict()
+    ip_group_id = str(args.get('ip_group_id', '')).strip()
+    check_url = f"/ipDestinationGroups/{ip_group_id}"
+    response_data = http_request('GET', check_url).json()
+    if response_data.get('id', 0) == 0:
+        raise Exception(f'RESOURCE NOT FOUND with ip_group_id. {ip_group_id}')
```

Comment:
Instead of raising an Exception, please use the `return_error` function call. 😉

Reply:
done.